composer.json

```diff
@@ -23,9 +23,7 @@
   "require"    : {
     "php": ">=5.4.0",
     "consolidation/robo": "~1",
-    "joomla/github": "~1.3",
-    "greencape/coding-standards": "~1",
```

Comment:
okay i am fine with that, still think we should remove the old metrics part from jorobo. And move it to the testing-repo.